.github/workflows/node.js.yml

```diff
@@ -47,5 +47,9 @@ jobs:
       - name: Lint checks
         run: pnpm lint
         working-directory: ./frontend
+
+      - name: Run depcheck
+        run: npx depcheck --ignore-bin-package
+        working-directory: ./frontend
         
       # future: add yarn build once build errors are resolved
```

Comment:
do we need to worry abt this todo @dwu359 @farisdurrani

Reply:
Yes, in prod we would usually publish the built package obtained from `pnpm next build` but that command is failing. We have to fix that before we can implement the todo

Reply:
but for this immediate action item, we don't need to worry about this for the scope of this PR? @farisdurrani